scripts/preview-gh-pages.js

```diff
@@ -0,0 +1,28 @@
+const { promisify } = require('util');
+const mv = promisify(require('mv'));
+const opn = require('opn');
+const fs = require('fs');
+const { createServer } = require('http-server');
+const portfinder = require('portfinder');
+portfinder.basePort = 4000;
+
+(async() => {
+  if (!fs.existsSync('docs/dist/seek-style-guide')) { // eslint-disable-line no-sync
+    await mv('docs/dist', 'docs/seek-style-guide', { mkdirp: true });
+    await mv('docs/seek-style-guide', 'docs/dist/seek-style-guide', { mkdirp: true });
```

Comment:
Assuming this is to simulate gh-pages?

Reply:
Yeah correct, it simulates the repo nesting on `https://seek-oss.github.io/` a.la the base href setting.